content/general-guidance.md

```diff
@@ -1,3 +1,127 @@
 ---
 title: "General guidance"
 ---
+
```

Comment:
This is fantastic... how do we represent the fact that the 'thing' to which the terms being changed are attached may be implemented by/standardized by others and there should be collaboration to ensure everyone is renaming *consistently* ?

Reply:
Hmmmmm.

Let me think on this. I think you're bringing up a good point that I need to rephrase for myself.